### DISCUSSION TITLE
Next Release with updated dependencies?

### DISCUSSION THREAD

**Discussion by PascalHonegger** [grpc/grpc-kotlin#336] [General]
Is there a roadmap of some sort of when a new release will be made available?
Our project currently experiences a bug from `grpc-java` and we manually have to override the version, we would really appreciate a release with the recently commited dependency updates

**Comment by jamesward** [comment]
I bumped the deps recently in the samples but I haven't bumped the main project deps due to some challenges.  I'll try to make some progress on that soon.
